client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -305,22 +306,32 @@ export const PatternLibrary = ( {
 									} }
 									value={ patternTypeFilter }
 								>
-									<ToggleGroupControlOption
-										className="pattern-library__toggle-option"
-										label={ translate( 'Patterns', {
-											comment: 'Refers to block patterns',
-											textOnly: true,
-										} ) }
-										value={ PatternTypeFilter.REGULAR }
-									/>
-									<ToggleGroupControlOption
-										className="pattern-library__toggle-option"
-										label={ translate( 'Page Layouts', {
-											comment: 'Refers to block patterns that contain entire page layouts',
-											textOnly: true,
-										} ) }
-										value={ PatternTypeFilter.PAGES }
-									/>
+									<Tooltip
+										text={ translate(
+											'A collection of blocks that make up one section of a page. (For example: an image gallery or contact form)'
+										) }
+									>
+										<ToggleGroupControlOption
+											className="pattern-library__toggle-option"
+											label={ translate( 'Patterns', {
+												comment: 'Refers to block patterns',
+												textOnly: true,
+											} ) }
+											value={ PatternTypeFilter.REGULAR }
+										/>
+									</Tooltip>
+									<Tooltip
+										text={ translate( 'A collection of patterns that form an entire page.' ) }
```

Comment:
Similar to my [previous comment](https://github.com/Automattic/wp-calypso/pull/89258/files#r1553726599):
```suggestion
										text={ translate( 'Collections of patterns that form an entire page' ) }
```